src/features/common/Layout/MyForestContextV2.tsx

```diff
@@ -0,0 +1,208 @@
+import {
+  FC,
+  createContext,
+  useContext,
+  useMemo,
+  useState,
+  useEffect,
+} from 'react';
+import { trpc } from '../../../utils/trpc';
+import { SetState } from '../types/common';
+import { ErrorHandlingContext } from './ErrorHandlingContext';
+import { useUserProps } from './UserPropsContext';
+import {
+  handleError,
+  APIError,
+  TreeProjectClassification,
+  CountryCode,
+  ProjectPurpose,
+  UnitTypes,
+} from '@planet-sdk/common';
+import { Point } from 'geojson';
+import {
+  ContributionStats,
+  MyContributionsMapItem,
+  MapLocation,
+} from '../types/myForestv2';
+
+interface ProjectInfo {
+  allowDonations: boolean;
+  classification: TreeProjectClassification;
+  country: CountryCode;
+  geometry: Point;
+  guid: string;
+  image: string;
+  name: string;
+  purpose: ProjectPurpose;
+  slug: string;
+  tpoName: string;
+  unitType: UnitTypes;
+}
```

Comment:
`unitType` can only be `tree` or `m2`, and purpose can only be `trees` or `conservation`.

Reply:
API response types - moving these to src/features/common/types/myForestv2.d.ts